docs: update ORAS intro (#518)
oras-project/oras-www: versioned_docs/version-1.3/index.mdx

Users seeking a generic registry client can benefit from the [ORAS CLI](./installation.mdx), while
developers can build their own clients on top of one of the [ORAS client libraries](./client_libraries/overview.mdx).

## Use cases

> ORAS enables a wide range of scenarios by treating OCI registries as generic artifact stores beyond container images. Some of the most common use cases include:

Reviewer: Something along the lines of (maybe):
```suggestion
It is not just managing container images that ORAS can do, it can treat OCI registries as a generic artifact store. Some of the most common use cases include:
```

Reviewer 2: What about something similar to the following

> ORAS usage extends beyond the management of container images as it treats OCI registries as a generic artifact store. Some of the most common use cases include:
